CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [1.5.10] - 2022-02-08
```

Comment:
Missing:

https://github.com/PyTorchLightning/pytorch-lightning/pull/11481 b8d2c65a372e5d01cfbeace72d6ebd024541ba86 Set the state before saving "last" or "none" checkpoints

Reply:
same here.

Reply:
Done!

In the future, you should either ping authors or change the merged PR milestone